runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPointsOverlay.java

```diff
@@ -80,6 +83,42 @@ public Dimension render(Graphics2D graphics)
 			.right(String.valueOf(client.getVar(Varbits.RAID_PARTY_SIZE)))
 			.build());
 
+		if (config.uniqueChance())
+		{
+			double chance = Math.min(totalPoints / 7125.0, 80.0);
+			double chance2 = Math.min((totalPoints - 570000.0) / 7125.0, 80.0);
```

Comment:
No need to calculate chance2/chance3 if the total points aren't even able to get you a drop. Would be better to move this inside their respective conditional blocks.